src/storefront/tweetreader.clj

```diff
@@ -0,0 +1,135 @@
+(ns storefront.tweetreader
+  (:require [storefront.drawing]
+            [storefront.helpers :refer :all]
+            [quil.core :as q]
+            [clojure.string :as str]
+            [storefront.textify :as textify])
+  (:import  [storefront.drawing Drawing]))
+
+(def tweeter "@smashingboxes")
+(def tweet "Designer @schombombadil talks to @designationio about making stuff at SB and smashing established UI patterns. http://sbox.es/1Qxe9OH")
```

Comment:
Yeah, I wanted your advice on how to do this since I know you are doing something similar for images. We should talk this week :)